Documentation/connectors/oidc.md

```diff
@@ -60,6 +60,12 @@ connectors:
     # or if they are acting as a proxy for another IDP etc AWS Cognito with an upstream SAML IDP
     # This can be overridden with the below option
     # insecureSkipEmailVerified: true 
+
+    # Some providers return this id tokens, when the claim you requested is not in the initial token
```

Comment:
nit: UserInfo is an OpenID Connect feature not OAuth2.
nit: no need to say this is disabled by default
nit: link to the UserInfo spec

Something like?

```
When enabled, the OpenID Connector will query the UserInfo endpoint for additional claims. UserInfo claims
take priority over claims returned by the IDToken. This option should be used when the IDToken doesn't contain
all the claims necessary to identify the user.
https://openid.net/specs/openid-connect-core-1_0.html#UserInfo
getUserInfo: true
```